nservicebus/upgrades/5to6.md

```diff
@@ -610,4 +610,38 @@ The BSON serializer build into the core has been removed. Use the [Newtonsoft se
 
 [MSMQ Subscription Authorization](/nservicebus/msmq/subscription-authorisation.md) is now done by the `SubscriptionAuthorizer` delegate at configuration time and not the `IAuthorizeSubscriptions` interface.
 
-snippet: 5to6-MsmqSubscriptionAuthorizer
\ No newline at end of file
+snippet: 5to6-MsmqSubscriptionAuthorizer
+
+## SQL Server Transport
```

Comment:
yeah. we can finesse the details later. but since other nugets can have multiple releases inside on major of the core, the only way is to have upgrade guides at the package level